serial-settings/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "serial-settings"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+embedded-storage = "0.3"
+miniconf = "0.9"
+menu = {path = "../../../rust-embedded-community/menu"}
```

Comment:
Currently need https://github.com/rust-embedded-community/menu/pull/15 to make this all work